miden/tests/integration/operations/field_ops.rs

```diff
@@ -315,6 +315,42 @@ fn pow2() {
     build_op_test!(asm_op, &[63]).expect_stack(&[1 << 63]);
 }
 
+#[test]
+fn ilog2() {
+    let asm_op = "ilog2";
+
+    build_op_test!(asm_op, &[1]).expect_stack(&[1, 0]);
+    build_op_test!(asm_op, &[2]).expect_stack(&[2, 1]);
+    build_op_test!(asm_op, &[3]).expect_stack(&[2, 1]);
+    build_op_test!(asm_op, &[4]).expect_stack(&[4, 2]);
```

Comment:
I think we should stress the rounding property a bit more here.

We have mostly perfect powers of 2. But we should assert that both ceiling and floor rounds works as expected. We test the `floor` with `3`, but I miss a ceiling test (that should be suppressed by floor as we always round down). I think a simple `5` case would do (as the log2 is below `2.5`).

Reply:
that should be covered by the proptest 

https://github.com/0xPolygonMiden/miden-vm/pull/874/files#diff-42baa4a6bb713a7887dc07b6d3c1436c2e6cacdde65fadb12d32f234f79dafd9R728

added the case with `5` for completeness nevertheless